components/BottomAppBar/src/MDCBottomAppBarViewController.h

```diff
@@ -0,0 +1,87 @@
+/*
+ Copyright 2016-present the Material Components for iOS authors. All Rights Reserved.
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+ http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+ */
+
+#import <UIKit/UIKit.h>
+
+#import "MaterialButtons.h"
+
+/** The position of the floating action button. */
+typedef NS_ENUM(NSInteger, MDCBottomAppBarFloatingButtonPosition) {
+  MDCBottomAppBarFloatingButtonPositionCenter = 0,
+  MDCBottomAppBarFloatingButtonPositionLeading = 1,
+  MDCBottomAppBarFloatingButtonPositionTrailing = 2
+};
+
+/**
+ A material design bottom bar.
+
+ The bottom bar is a bar docked at the bottom of the screen. A floating action button is provided
+ for a primary action.
+ */
+@interface MDCBottomAppBarViewController : UIViewController
+
+/**
+ Is the floating button on the bottom bar visible.
+ Default is YES.
+ */
+@property(nonatomic, assign, getter=isFloatingButtonVisible) BOOL floatingButtonVisible;
+
+/**
+ The floating button on the bottom bar. This button is exposed for customizability.
+ */
+@property(nonatomic, strong, readonly, nonnull) MDCFloatingButton *floatingButton;
+
+/**
+ The position of the floating action button.
+ Default is MDCBottomAppBarFloatingButtonPositionCenter.
+ */
+@property(nonatomic, assign) MDCBottomAppBarFloatingButtonPosition floatingButtonPosition;
+
+/**
+ The embedded view controller that appears behind the bottom app bar. This content is part of the
+ application and not visually connected to the bottom app bar.
+ */
+@property(nonatomic, strong, nonnull) UIViewController *viewController;
```

Comment:
Since we have 2 VCs that are children here, I'd love for this one to have a more specific name like content one does.

Reply:
I had previously named this contentViewController, and the second VC bottomAppBarViewController. However, Jeff suggested for consistency to call the embedded application VC, viewController, to be consistent with other components. In other components the content area VC contained in the controller is called contentViewController. In short, this naming scheme is more consistent with other components.

Reply:
Do any other components have a `viewController` and another similar property? (Just asking. Not being passive aggressive. I actually don't know.)

Reply:
MDCFlexibleHeader has a contentViewController that has the content displayed behind the header. MDCTabBar has its viewControllers property for the view controllers associated with the content for each tab.